### ISSUE TITLE
Add images in LaTeX?

### ISSUE THREAD

**Issue by RomeoDev777** [michael-brade/LaTeX.js#111]
I couldn't find in the examples how I can add images. Is it possible to add an image such as PNG?
There's a picture command but I guess it's only for drawing and not displaying images.

**Comment by michael-brade** [comment]
Yes, you are right. I have not implemented `\includegraphics` yet, but it is quite high on my TODO list. I want to fix an issue with \put and the whitespace bugs first, then I'll do the graphics stuff.

**Comment by RomeoDev777** [comment]
Thanks, that would be an awesome feature to have.
